tests/harvest/test_harvest_cli.py

```
@@ -1,34 +1,42 @@
+
+
+def test_help(capfd: pytest.CaptureFixture) -> None:
+    help_line: str = "--help"
+    with pytest.raises(SystemExit) as caught:
+        _main(*help_line.split())
+    assert caught.value.code == 0
+    assert caught.type is SystemExit
+    out, err = capfd.readouterr()
+    assert len(out) > 0
+    assert "usage: " in out
+    assert "-c" in out and "--config" in out
```

Ruff

Assertion should be broken down into multiple parts

Break down assertion into multiple parts

(PT018)